processing/src/main/java/org/apache/druid/segment/incremental/ParseExceptionHandler.java

```diff
@@ -76,27 +86,51 @@ public void handle(@Nullable ParseException e)
       rowIngestionMeters.incrementProcessedWithError();
     } else {
       rowIngestionMeters.incrementUnparseable();
-    }
+      if (emitter != null) {
+        DateTime timeOfException = new DateTime(e.getTimeOfExceptionMillis());
```

Comment:
To enforce UTC timestamp:
```suggestion
        DateTime timeOfException = DateTimes.utc(e.getTimeOfExceptionMillis());
```